backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/AbstractAnalyticsManager.java

```diff
@@ -123,16 +172,160 @@ public AbstractAnalyticsManager(String apiEndpoint, String workspaceId, String m
 
     stackId = workspace.getAttributes().get("stackName");
     factoryId = workspace.getAttributes().get("factoryId");
+    setFactoryVariables(endpoint, workspaceConfig);
+    osioSpaceId = workspace.getAttributes().get("osio_spaceId");
```

Comment:
would be nice to remove this outdated + Hosted Che specific property

Reply:
I wasn't sure if it was required or not.  If it is no longer needed, I can remove it

Reply:
yeah, this attribute one is outdated